README.md

```diff
@@ -1,82 +1,177 @@
+  2. Click on the "Shared Drivers" section and choose the driver to be shared.
+
+     ![Shared driver](./docs/images/shared-drivers.png)
+
+- Install Docker Desktop for Mac
+
+  Follow the [installation guide](https://docs.docker.com/docker-for-mac/install/) and make sure to enable driver sharing for Docker to access your local folder:
+    *[TBD for Mac]*
+
+- Get Docker for Linux
+
+  Linux is a highly variable environment and the large number of server, container, and desktop distributions can make it difficult to know what is supported. Please check this [tutorial](https://code.visualstudio.com/docs/remote/linux) for Linux support.
+
+## Usage
+
+Depending on the device platform you choose, follow the resources below to develop on it.
+
+### Embedded Linux <sup>preview</sup>
+
+To simplify the cross compiling toolchain, Azure IoT Device SDK and dependencies setup and configuration, Device Workbench put all these components in the container. All cross-compiling works happen in it. Here is the quick start of using it to develop and compile a simple device app written in C and running on [Raspberry Pi 3 Model B+](https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/) to send telemetry data to Azure IoT Hub.
+
+#### Create new project
+
+1. Within VS Code, press `F1` to open the command palette, then type and select **Azure IoT Device Workbench: Create Project...**.
+
+2. Select the working folder (only for the first time) and enter the name of your project.
+
+3. Select **Embedded Linux** from device platform list.
 
-* [Get Started](./docs/iot-button/teXXmo_IoT_button_get_started.md)
+4. Select the dev container based on your target device, in this sample, select **32-bit Armv7 Cortex-A** for Raspberry Pi 3 Model B+ running Raspbian.
 
-### Raspberry Pi
+5. For the first time, it takes around 1 to 3 mintues depending on your Internet speed to download and prepare the dev container. You can click the **details** link on the notification for the progress:
+
+     ![Prepare Dev Container](./docs/images/prepare-dev-container.png)
+
+6. Once the dev container is ready, VS Code will open a new window with project files the sample code.
+
+#### Compile the code
+
+1. The `iothub_sample.c` under the `src` subfolder is the source file contains the logic. You can modify or add your own code in it.
+
+    ![iothub_sample.c](./docs/images/iothub_sample.png)
+
+2. To compile the code, press `F1`, type and select **Azure IoT Device Workbench: Compile Device Code** from the command palette.
+
+3. The cross-compiling of the code happens in the dev container. Once its done, it pops up the notification.
+    ![Compile success](./docs/images/compile-success.png)
+    
+4. The compiled binary file is located in `.build` folder.
+
+#### Upload to target device
+
+1. Before upload the executable binary file to the target device, please make sure:
+
+   - The running OS on the device is correct (e.g. Raspbian running on Raspberry Pi 3 Model B+).
```

Comment:
Make sure the OS is 32-bit. Only on 32-bit OS target architecture can the binary be run.